### PR TITLE
Add new pillars to be more modular

### LINKED ISSUE #29: The file init.sls contains obsolete repo
I really don't feel the state `aptly_repo` belongs in init.sls, that will run in jessie and install the wrong version of aptly. We could wrap in a condition, but I lean towards leaving that out altogether. I feel if someone needs a repo they can add that in a state before the aptly formula is run, no? And it could very well be their own private deb repo anyway.

### PR DESCRIPTION
**Pull request by aanriot**
Hello,

This PR allows to disable package installation. Indeed, you may want to setup the *APT* repository with the self-titled formula, install the package with the *packages* formula, etc.

It should not affect existing setups.

Fixes #29 .

Thanks.

### CODE REVIEW

File: aptly/init.sls [hunk 1]
```diff
@@ -7,28 +12,35 @@ aptly_repo:
...
-      - pkg: aptly
+      - pkg: aptly_packages
+{% endif %}
 
-aptly:
+{% if install_packages %}
+aptly_packages:
   pkg.installed:
-    - name: aptly
+    pkgs:
```
Comment (aboe76) [inline aptly/init.sls]: can this be set in a map.jinja style so we can differentiate between OS?

Comment (aanriot) [reply aptly/init.sls]: Done @aboe76 .

**Comment by aanriot** [comment]
I have resolved the conflicts a few minutes ago @aboe76 .

**Comment by aboe76** [comment]
@aanriot thanks for this.

**Pull request merged**